ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.collection.tsx

```diff
@@ -53,16 +53,36 @@ const ResultGroup = styled.div`
   }
 `
 
+const DidYouMeanContainer = styled.div`
+  text-align: center;
+`
+
+const ShowMore = styled.a`
+  padding: 0.25rem;
+  font-size: 12px;
```

Comment:
We should remove any references to px and use rem, that way things scale in all the spacing modes.  The conversion is to divide by 16px.  So 12px would be 0.75rem.